tests/run-pass/used_underscore_binding_macro.rs

```diff
@@ -1,12 +1,9 @@
-
-
-
 #[macro_use]
 extern crate serde_derive;
 
 /// Test that we do not lint for unused underscores in a `MacroAttribute`
 /// expansion
-#[deny(used_underscore_binding)]
+#[deny(clippy::used_underscore_binding)]
 #[derive(Deserialize)]
```

Comment:
I get a weird error here:
```bash
error: proc-macro derive panicked
 --> tests/run-pass/used_underscore_binding_macro.rs:7:10
  |
7 | #[derive(Deserialize)]
  |          ^^^^^^^^^^^
  |
  = help: message: called `Result::unwrap()` on an `Err` value: "failed to parse derive input: \"/// Test that we do not lint for unused underscores in a `MacroAttribute`\\n/// expansion\\n#[deny(clippy::used_underscore_binding)]\\nstruct MacroAttributesTest {\\n    _foo: u32,\\n}\""
```

That's because somehow `syn` fails to parse the new `tool_lints`... 
See [serde](https://github.com/serde-rs/serde/blob/4e54aaf7963c3580cc50b56842949b0ce6b3a997/serde_derive/src/lib.rs#L92) and [syn](https://github.com/dtolnay/syn/blob/209a78c8629c3aab771e2d6fd3c4f7b84d75825f/src/lib.rs#L650-L659)

Reply:
we just need to update our dependency version requirement to the most recent one, then it'll work